pkg/apis/cartographer/v1alpha1/workload_test.go

```diff
@@ -1767,30 +1800,6 @@ func TestWorkloadSpec_MergeGit(t *testing.T) {
 				Subpath: "my-subpath",
 			},
 		},
-	}, {
-		name: "update to git source deleting subpath",
```

Comment:
why were these deleted? they do not apply anymore?

Reply:
As the `Merge` is now doing merge instead of a replacement, then it does not apply now.